packit/cli/builds/copr_build.py

```diff
@@ -18,7 +18,7 @@
 
 
 @click.command("in-copr", context_settings=get_context_settings())
-@click.option("--nowait", is_flag=True, default=False, help="Don't wait for build")
+@click.option("--wait/--no-wait", default=False, help="Wait for the build to finish")
```

Comment:
I'm not sure if you're intentionally switching the default :P (waiting → no waiting)

I don't have any issue with that though

Reply:
/o\ my bad, will fix to keep consistency

thank you for noticing!

Reply:
also koji builds were flipped